musicAlbum/music_album.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify, :publish_date
+
+  def initialize(on_spotify, author, label, published_date)
+    super(author, label, published_date)
+    @on_spotify = on_spotify
+  end
+
+  def can_be_archived?()
+    true if super() && @on_spotify
+  end
+end
```

Comment:
- [ ] As it is mentioned in the project requirements, kindly include the unit tests for all the implemented methods for the music album class. This will ensure that you meet the project requirements.

![image](https://user-images.githubusercontent.com/86318284/213567791-97dbf63f-8662-4458-b5e2-561c859aaa9b.png)